disable OSC 8 sequences as our less(1) has issues with it.

Index: filters/colorize.c
--- filters/colorize.c.orig
+++ filters/colorize.c
@@ -423,6 +423,7 @@ static inline size_t print_notabs(const char *in, size
 }
 
 static void print_osc8(const char *url, size_t len, size_t id, bool email) {
+#if 0
 	print("\x1b]8;");
 	if (url != NULL) {
 		printf("id=colorize-%lu;", id);
@@ -435,6 +436,7 @@ static void print_osc8(const char *url, size_t len, si
 		print(";");
 	}
 	print("\x1b\\");
+#endif
 }
 
 static void diff_chunk(const char *in)
